projects/plugins/jetpack/extensions/blocks/subscriptions/panel.js

```diff
@@ -139,14 +144,16 @@ function NewsletterPrePublishSettingsPanel( {
 			className="jetpack-subscribe-pre-publish-panel"
 			icon={ <JetpackLogo showText={ false } height={ 16 } logoColor="#1E1E1E" /> }
 		>
-			<NewsletterAccessPrePublishSettings
-				accessLevel={ accessLevel }
-				setPostMeta={ setPostMeta }
-				socialFollowers={ socialFollowers }
-				emailSubscribers={ emailSubscribers }
-				paidSubscribers={ paidSubscribers }
-				showMisconfigurationWarning={ showMisconfigurationWarning }
-			/>
+			{ isModuleActive && (
```

Comment:
Not sure if we need `isModuleActive` here when we have the check above.

Reply:
I think the GitHub UI makes this a bit confusing. The check above is for another component, but the GitHub UI collapsed lines 65 to 144, so we do not see that.